File: reto9.py
expresion = "{[3+4(1/2)]*2}"


def verificarExpresion(text):
    cont = 0
    for i in text:
        if i == "{" or i == "(" or i == "[":
            cont += 1
        elif i == "}" or i == ")" or i == "]":
            cont -=1
    return cont == 0


print(verificarExpresion(expresion))


    

File: reto9.js
let expresion = "{[3+4(1/2)]*2}"

function verificarExpresion(text){
    let cont = 0
    for (let i of text) {
        if(i == "{" || i == "[" || i == "("){
            cont +=1
        }else if(i == "}" || i == "]" || i == ")"){
            cont -=1
        }
    }
    console.log(cont)
    return (cont == 0)
}


console.log(verificarExpresion(expresion))

/*arreglar los retos 9*/